### DISCUSSION TITLE
Fertilizer and subtypes

### DISCUSSION THREAD

**Discussion by Rook982** [sunflower-land/sunflower-land#1143] [Ideas]
I want to start a discussion around Fertilizers, we know that it was planned to be the product of pigs which will be "Manure", which will then a component for Fertilizer. Fertilizers will then be used as a consumable buff to the fields, crops that have been fertilized will grow 2x faster and be worth 2x more. But that got me thinking that we could expand upon it by making various types of fertilizer. Here is what I have in mind.

Speed Fertilizer - Speeds up growing time of crops when applied. Decreases selling price by 20%* 
Crafting Requirements - Sunflower, Manure

Quality Fertilizer - Doubles selling price of crops when applied. Lengthens growing time by 50%*
Crafting Requirements - Radish, Manure

Quantity Fertilizer - Doubles yield of crops when applied. Lengthens growing time by 100%, Decreases selling price by 10%*
Crafting Requirements - Potato, Cauliflower, Manure

*I am not too informed on what is the proper percentages and for the caveats so I hope the number I put in is reasonable and could be changed by the developers to the proper amount when implemented. 

These will allow fertilizers to have more variety instead of the one and done 2x multiplier. It also gives another utility for different crops.

Here are some fertilizer ideas that are a bit weird but kinda fun.

Mutant Fertilizer - Increases chance to grow a Mutant Crop
                           - When used on wheat it becomes Weird Wheat, which can then be used to feed the chickens for increased chance of                   Mutant chicken or Animal.
                           
 Golden Fertilizer - Chance to grow a golden crop. 
 Crafting Requirements - Gold and Manure.
 
 Armament Fertilizer - Chance to grow a crop weapon or armor piece.
 
 This is all I could come up with for now, what do you think? I don't have the skills to create the graphics and images but hope this could spark an idea for the future mechanic of fertilizer.

**Comment by Yougottabekiddingme123** [comment]
I like ther overall  concept. I think it would add some depth to the game beyond the basic planting and harvesting. ❤

**Comment by azarzaza** [comment]
A Composter NFT, on the other hand, may create a wide range of fertilizer based on what we feed it.
Putting a specific amount of crop and animal manure, for example, will result in higher-quality fertilizer.
